### DISCUSSION TITLE
Welcome to the Patrol Group

### DISCUSSION THREAD

**Discussion by Zitchas** [endless-sky/endless-sky#8490] [Team Posts]
I'm still learning just what we can do with teams and organizations on GitHub, and this is going to be the first trial of such.

Everyone in this group has "triage" permissions on the main Endless-Sky repository, which means, among other things, that you can do things like label issues and PRs; as well as a range of moderation style options and abilities on the github discussion board. You also get access (apparently) to this nifty little discussion board. 

I don't know if it is actually private or not (we'll have to test that), and I'd venture that it's best to assume that it's not actually private; but in any case probably a good place to discuss stuff related to triage, reviewing, and github moderation.

edit: While each of you have been added directly to the main ES repository, I have added you all collectively as a team to the high-dpi repository with the same triage permissions.

**Comment by Terin** [comment]
Sounds good! And yeah, looks like everything under the Teams tab/URL is completely private unless you're part the teams there, so that should be covered.

**Comment by Hurleveur** [comment]
Does anyone have any ideas why in github we can't resolve our own comments?
Its pretty weird that you have to tell the PR owner that they can mark it resolved, anyway I wonder if that's something that could be added, or is it a 'all of nothing' situation?

edit: also ty for the trust you have given us, I will try to make good use of any new responsibilities

**Comment by Zitchas** [comment]
I can't change permissions that are predefined as part of the set roles that github gives us (that's reserved for *paid* enterprise organizations that can make their own roles), but we can definitely look into that.

It makes sense to me that whoever makes a comment should also be able to mark it resolved. After all, the original commenter is probably best positioned to say whether or not their comment, question, or suggestion has been resolved.

**Comment by MasterOfGrey** [comment]
Yeah that might be part of the repository settings potentially - samrocketman has been talking about those quite extensively of late.

**Comment by quyykk** [comment]
Just noticed this, this is cool! I think tehhowch also wanted to create different teams, so nice that we're able to start. This team is "public", so anybody that is part of the org can see it (but anybody outside can't). You can also mark it private, but I don't really see why you should.

**Comment by Zitchas** [comment]
Yeah, I saw that when I created it. This role (and associated team) is intentionally a public-facing thing, so public seemed appropriate.

**Comment by Zitchas** [comment]
@EjoThims and @tibetiroka 

Welcome to the newest members of the Patrol Group. This discussion room is probably roughly the equivalent of the Reviewer channel on the discord, except, obviously, it's not on Discord. So it's accessible to those without Discord access; and likewise is independent of the Discord community & infrastructure.

**Comment by EjoThims** [comment]
Does anyone know why GitHub decided to spam me with notifications about every post in here today, despite them all being months old? 😵

**Comment by Zitchas** [comment]
Github is actually retiring the internal teams chat rooms. It was either "have it all get dumped into discussions in a couple months", or just pull the trigger and dump it in here now. I picked the later to avoid having surprises later.

**Reply by EjoThims** [reply]
I didn't realize. That makes sense, yea.

**Comment by xX-Dillinger-Xx** [comment]
I'm not sure if your aware but this area is not private. I'm nobody, and I can see it.

**Reply by Zitchas** [reply]
It was initially private, and then at somepoint GitHub just decided to merge it all into regular discussions. It is known, but I appreciate the comment/heads up anyway. (see my post https://github.com/endless-sky/endless-sky/discussions/8490#discussioncomment-5185753 for a statement to this effect.

**Reply by xX-Dillinger-Xx** [reply]
Yeah, read I that. It just wasn't clear if it was to remain private or not. I don't even know if that's a thing in 'discussions'. I guess I know why nobody uses it.  Cheers

**Reply by Zitchas** [reply]
Discussions are a fairly recent thing that GitHub implemented, and, while useful, are largely secondary to issues for most uses. They're still nice to have, though. GitHub made the decision to merge the private discussions into the public ones while we were still implementing the reviewer system, and as such never really got used much on that side of things

**Reply by xX-Dillinger-Xx** [reply]
I actually prefer the 'discussions' area, it allows better communication. Easier to reply to individual posts within the topic.  Just, like this one. 
After all the discussion is mostly finished, then it can move to an issue or PR.  Just my opinion.
